contracts/receivers/aave-v2-receiver/main.sol

```diff
@@ -56,6 +56,14 @@ contract MigrateResolver is Helpers, Events {
                     TokenInterface(wmaticAddr).deposit{value: address(this).balance}();
                 }
             }
+
+            if (_token == wethAddr) {
+                TokenInterface wethPosContract = TokenInterface(wethPosAddr);
+                if (wethPosContract.balanceOf(address(this)) > 0) {
+                    WETHTokenInterface(wethAddr).deposit(address(this), abi.encode(wethPosContract.balanceOf(address(this))));
```

Comment:
From [Matic WETH](https://explorer-mainnet.maticvigil.com/address/0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619/contracts) #L1502
`Should be callable only by ChildChainManager`

I don't think will work. ChildChainManager is [0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa](https://explorer-mainnet.maticvigil.com/address/0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa/transactions)

Reply:
Yess! I will remove this.